test/fixtures/ledger_transaction_fixtures.js

```diff
@@ -0,0 +1,287 @@
+'use strict'
+
+// NPM dependencies
+const path = require('path')
+const lodash = require('lodash')
+
+// Global setup
+const pactBase = require(path.join(__dirname, '/pact_base'))
+const pactRegister = pactBase()
+
+const buildChargeEventWithDefaults = (opts = {}) => {
+  const chargeEvent = {
+    type: opts.type || 'PAYMENT',
+    state: buildChargeEventStateWithDefaults(opts.state),
+    amount: opts.amount || 100,
+    updated: opts.updated || '2018-05-01T13:27:00.063Z'
+  }
+  if (opts.refund_reference) {
+    chargeEvent.refunds_reference = opts.refund_reference
+  }
+  if (opts.submitted_by) {
+    chargeEvent.submitted_by = opts.submitted_by
+  }
+  return chargeEvent
+}
+
+const buildChargeEventStateWithDefaults = (opts = {}) => {
+  let state
+  if (opts.status === 'failed') {
+    state = {
+      status: opts.status,
+      finished: true,
+      code: opts.code || 'P0010',
+      message: opts.message || 'Payment method rejected'
+    }
+  } else {
+    state = {
+      status: opts.status || 'started',
+      finished: opts.finished || false
+    }
+  }
+
+  return state
+}
+
+const buildTransactionCreatedDetailsWithDefaults = (opts = {}) => {
```

Comment:
As these three methods (`buildTransactionCreatedDetailsWithDefaults`, `buildTransactionDetailsWithDefaults` and `buildTransactionSearchResultWithDefaults`) are all 'builders' would it be possible to have one method that can then add details as needed? 

For example there could be an `opts.includeSettlementSummary` which would only add that part of the object is it was true. 

This might reduce the lines of code needed for the fixture and mean the fixture is only defined once instead of three different times.